### PR TITLE
Improve documentation comments

### PR DESCRIPTION
**Pull request by xeruf**
Please don't create an epic fuss and let conflicts pile up. These changes were done while I worked on the Macros feature and capture insights and code quality improvements that would have to be done either way at some point.
@Holzhaus

### CODE REVIEW

**Review by Holzhaus — CHANGES_REQUESTED** [review]
Thanks, the doxygen-style comments are definitely useful. I left some comments.

File: src/track/track.h [hunk 1]
```diff
@@ -102,25 +102,21 @@ class Track : public QObject {
     // Get number of channels
     int getChannels() const;
 
-    // Get sample rate
```
[the remark this answers is not in the document]
Comment (xeruf) [reply src/track/track.h:105]: It being about the _number_ of channels is only deductible from seeing the signature or the docs
But I can of course also adjust it.

File: src/recording/recordingmanagerbase.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include "recordingmanagerbase.h"
+
+void RecordingManagerBase::slotSetRecording(bool record) {
```
[the remark this answers is not in the document]
Comment (xeruf) [reply src/recording/recordingmanagerbase.cpp:3]: Right, recently I was advised to prefer squashing so I didn't bother

File: src/track/track.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (Holzhaus) [reply src/track/track.h:105]: Ah right, that makes sense.

File: src/track/track.cpp [hunk 3]
```diff
@@ -345,6 +345,16 @@ void Track::adjustReplayGainFromPregain(double gain) {
     }
 }
 
+void Track::analysisFinished() {
```
[the remark this answers is not in the document]
Comment (xeruf) [reply src/track/track.cpp:348]: The benefit is the grouping of related methods and sometimes ordering alignment to the header file, making it much easier to find your way around a file.

File: src/track/track.h [hunk 4]
```diff
@@ -102,25 +102,21 @@ class Track : public QObject {
...
     mixxx::audio::SampleRate getSampleRate() const;
 
-    // Sets the bitrate
     void setBitrate(int);
-    // Returns the bitrate
     int getBitrate() const;
-    // Returns the bitrate as a string
```
[the remark this answers is not in the document]
Comment (xeruf) [reply src/track/track.h:112]: I would expect "320", if that is not so it should be documented yeah.

File: src/recording/recordingmanagerbase.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (Holzhaus) [reply src/recording/recordingmanagerbase.cpp:3]: I don't know who said that but squashing/amending only makes sense for commits working on the same topic, not unrelated commits. We do prefer squashing for stuff like controller mappings where people have a bunch of commits with useless commit messages like "update mapping", but not in other cases.

**Comment by ronso0** [comment]
Stale since 2022, many conflicts, **xeruf** vanished after GSoC, closing this.

**Pull request closed without merging**